### ISSUE TITLE
why the inference need to take so much time, one image need more 1 minute?

### ISSUE THREAD

**Issue by wmmin** [USTC-JialunPeng/Diverse-Structure-Inpainting#22]

**Comment by USTC-JialunPeng** [comment]
Indeed, the inference time of our autoregressive model is long compared to GAN-based or VAE-based models. The majority of the time is spend iteratively running the structure generator for 32x32 times. Please refer to the [Supplementary Material](https://openaccess.thecvf.com/content/CVPR2021/supplemental/Peng_Generating_Diverse_Structure_CVPR_2021_supplemental.pdf) for details on `Inference Time`.
